modules/registry-configuring-storage-baremetal.adoc

```diff
@@ -57,4 +57,16 @@ $ oc get clusteroperator image-registry
 //type: Degraded
 //----
 
-​​
+[Note]
+=====
+If the storage type is `emptyDIR`, the replica number can not be greater than `1`.
+If the storage type is NFS, and you want to scale up the registry Pod by setting
```

Comment:
Should "NFS" be shown as a literal value like how `emptyDIR` is? Not sure if it's `nfs` or whatnot.